cpp/ql/src/semmle/code/cpp/models/implementations/Printf.qll

```diff
@@ -5,9 +5,10 @@ import semmle.code.cpp.models.interfaces.FormattingFunction
  */
 class Printf extends FormattingFunction {
   Printf() {
-    this instanceof TopLevelFunction and 
+    this instanceof TopLevelFunction and
     (
       hasGlobalName("printf") or
+      hasGlobalName("__builtin_printf") or
```

Comment:
How confident are we that `__builtin_printf` (and `__builtin_sprintf`, `__builtin_snprintf`) will match the expected patterns (for format parameter position, wideness)?

Reply:
I'm just going by the GNU extensions, where the parameters and wideness (narrow) match.

Reply:
I think I'm OK with that.